.changes/next-release/bugfix-AWSSDKforJavav2-1b88003.json

```diff
@@ -0,0 +1,5 @@
+{
+    "category": "AWS SDK for Java v2", 
+    "type": "bugfix", 
+    "description": "Fixed a minor error in ByteArrayAsyncRequestBody.java and resolved the issue of redundant requests."
```

Comment:
Can we update it to something like 

Fixed the issue where  `ByteArrayAsyncRequestBody` can send duplicate requests when another request comes in at the same time the subscription completes.